packages/itwin/grouping-mapping-widget/src/widget/components/GroupsView.tsx

```diff
@@ -98,10 +107,10 @@ export const GroupsView = ({
       </div>
       {alert}
       <div className='gmw-groups-border' />
-      {!!numberOfVisualizedGroups &&
+      {!!visualizedGroups &&
         <div className="gmw-group-progress-bar">
           <ProgressLinear
-            value={25 + (numberOfVisualizedGroups / groups.length * 65)}
+            value={25 + (visualizedGroups.numberOfVisualizedGroups / visualizedGroups.totalNumberOfVisualization * 65)}
```

Comment:
What are these magic numbers in this formula?

Reply:
Made it more clear in the code.